express/parsers/molecule.py

```diff
@@ -0,0 +1,113 @@
+from io import StringIO
+import os
+from ase.io import read, write
+import ase
+import rdkit
+from rdkit import Chem
```

Comment:
I don't think you need this import b/c you are using `rdkit.Chem` in https://github.com/Exabyte-io/express/pull/89/files#diff-4d439e5b78699b91fec575ace9a2bb6d4aa58b9ad3c8aa1b9a24a8ce7821850bR107 already

Reply:
oddly enough, without the `from rdkit import Chem` line, that code on line 107 fails, when calling `rdkit.Chem` I get the error ```inchi_key_val = rdkit.Chem.inchi.InchiToInchiKey(self.inchi)\nAttributeError: module 'rdkit' has no attribute 'Chem'\n"```

So i kept the `from rdkit import Chem` line and changed the line on 107 to `Chem` instead of `rdkit.Chem`, which works.